modules/002-deckhouse/hooks/enable_extended_monitoring_test.go

```diff
@@ -53,12 +53,12 @@ metadata:
 			Expect(f).To(ExecuteSuccessfully())
 
 			annotation := f.KubernetesGlobalResource("Namespace", "d8-system").
```

Comment:
You are referring to label, but there will be no `flant.com` label. At least the variable name could be actualized

```suggestion
			label := f.KubernetesGlobalResource("Namespace", "d8-system").
```

Reply:
Remove the migration!